CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 ### Changed
 
+- [BREAKING CHANGE] Removed the `getStarkPublicKey` method
```

Comment:
Can you also mention removed `getKeyPairFromPrivateKey` method, need to know when it would release